aip/client-libraries/4222.md

```diff
@@ -33,14 +33,18 @@ rpc CreateTopic(CreateTopicRequest) {
 In this case, the applicable variable is `parent`, and it refers to the
 `parent` field in `CreateTopicRequest`. When the user provides an instance of
 `CreateTopicRequest` to the method (or once the client library has built it, in
-the case of method overloads), the client library should then extract the key
-and value, URL-encode them, and append them to the `x-goog-request-params`
-header. Much like URL parameters, if there is more than one key-value pair, the
-`&` character is used as the separator.
+the case of method overloads), the client library **must** extract the key
```

Comment:
I feel like this `must` doesn't actually need to be emphasized. It could be rewritten as `the client library extracts the key and value...`, because "duh" the key & value have to be extracted. WDYT?